### PR TITLE
Mention that Locales and Translations section is specific to glibc

### PR DESCRIPTION
**Pull request by grtcdr**
This pull request closes issue #821.

### CODE REVIEW

**Comment by abenson** [comment]
I'd explain that `musl` doesn't support locales.

**Comment by grtcdr** [comment]
I'll do that right away.

**Comment by grtcdr** [comment]
Would you consider https://wiki.gnome.org/Initiatives(2f)Wayland(2f)Applications.html a reasonable replacement for the currently broken GNOME link?

Here's an archived version of that URL for comparison: https://web.archive.org/web/20240620195313/https://wiki.gnome.org/Initiatives/Wayland/GTK%2B

**Comment by classabbyamp** [comment]
setting `LANG` should still work on musl for applications, iirc

**Comment by grtcdr** [comment]
Does that environment variable matter though? https://wiki.musl-libc.org/environment-variables doesn't list it.

**Comment by grtcdr** [comment]
@classabbyamp: I phrased things differently to take into account the fact that `musl` does indeed support `LANG`. It would be great if the musl documentation had anchors for the different headings because then we could reference that section.

**Comment by classabbyamp** [comment]
musl itself doesn't support `LANG`, but most applications will use `LANG` to set the language, even in the absence of system locale support (like on musl)

**Comment by classabbyamp** [comment]
`/etc/locale.conf` is just sourced by a `profile.d` snippet (`/etc/profile.d/locale.sh`) so it should apply regardless

**Comment by grtcdr** [comment]
I understand, I said it wasn't required because it apparently isn't. That statement does not negate yours, though.

File: src/config/locales.md [hunk 1]
```diff
@@ -14,7 +16,8 @@ To enable a certain locale, un-comment or add the relevant lines in
 
 ## Setting the system locale
```
Comment (classabbyamp) [inline src/config/locales.md:17]: this heading is inaccurate, it's for setting the system *language*

File: src/config/locales.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # Locales and Translations
 
-For a list of currently enabled locales, run
+You can use the following command, which is specific to `glibc` installations of
+Void (given that `musl` does not support locales), to see a list of currently
+enabled locales:
```
Comment (classabbyamp) [inline src/config/locales.md:5]: personally, I think this is better as an introductory sentence/paragraph, so something like this

```suggestion
glibc supports setting the system locale, while musl does not. Both support setting the language for applications.

## Listing locales

For a list of currently enabled locales, run:
```

File: src/config/locales.md (hunk 1 above)
Comment (grtcdr) [reply src/config/locales.md]: Fixed.

File: src/config/locales.md (hunk 2 above)
Comment (grtcdr) [reply src/config/locales.md]: I like that too, fixed.

File: src/config/locales.md [hunk 3]
```diff
@@ -12,9 +17,10 @@ To enable a certain locale, un-comment or add the relevant lines in
 `/etc/default/libc-locales` and [force-reconfigure](../xbps/index.md) the
 `glibc-locales` package.
 
-## Setting the system locale
+## Setting the system language
 
-Set `LANG=xxxx` in `/etc/locale.conf`.
+Set `LANG=xxxx` in `/etc/locale.conf`, this applies to `glibc` and `musl`
+although it's not required by the latter.
```
Comment (classabbyamp) [inline src/config/locales.md:23]: it is required for glibc, if you want the language to change for most applications

Comment (grtcdr) [reply src/config/locales.md:23]: I never said it wasn't though, did you mean to say musl instead of glibc?

Comment (classabbyamp) [reply src/config/locales.md:23]: i read latter as former, but `LANG` is required for setting the system language on both libcs

Comment (grtcdr) [reply src/config/locales.md:23]: I do that too sometimes, I'll remove that part altogether.

Comment (grtcdr) [reply src/config/locales.md]: Okay, done.

File: src/config/locales.md [hunk 4]
```diff
@@ -1,6 +1,11 @@
 # Locales and Translations
 
-For a list of currently enabled locales, run
+`glibc` supports setting the system locale whereas `musl` does not, both support
```
Comment (classabbyamp) [inline src/config/locales.md:3]:
```suggestion
`glibc` supports setting the system locale whereas `musl` does not; both support
```
nit

**Review by classabbyamp — COMMENTED** [review]
please squash your commits into 1 with all these changes, then this should be good to go

**Comment by grtcdr** [comment]
Okay, all set :)

**Pull request merged**